ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-mgdapi-1595-cluster-scoped-3scale.yaml

```diff
@@ -101,10 +101,14 @@ tests:
           cpu: 100m
     workflow: generic-claim
 - as: multitenant-rhoam-e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.11"
```

Comment:
Was this not meant to be added in a different PR. Or is it a case that the branch needs a rebasing

Reply:
Yeah, the previous PR was only for master. Branches would need to be updated also if they want to use cluster pools also, so I thought may aswell keep them consistent